sdk/src/main/java/io/radar/sdk/RadarForegroundService.kt

```diff
@@ -45,29 +86,31 @@ class RadarForegroundService : Service() {
     }
 
     private fun startForegroundService(extras: Bundle?) {
+        started = true
         val manager = getSystemService(NOTIFICATION_SERVICE) as NotificationManager
-        manager.deleteNotificationChannel("RadarSDK")
-        var id = extras?.getInt("id") ?: 0
+        manager.deleteNotificationChannel(RADAR_CHANNEL)
+        var id = extras?.getInt(KEY_FOREGROUND_SERVICE_ID) ?: 0
```

Comment:
could these two lines be

`val id = extra?.getInt(KEY_...) ?: NOTIFICATION_ID`

or would it be possible this key could be 0?

Reply:
Good catch - been learning not to refactor more than necessary, but this seems like a pretty small improvement.